README.md

```diff
@@ -20,47 +20,78 @@
 * Polygon or polyline used for spatial 'coordinates' of a timeseries variable.
 * Mimic well known text style for encoding multipolygons with holes and multilines.
 
-## Important Elements and Structure
-
-For discussion, the structure and attributes below should be what's needed. This is using a rectangular array for the data and an indexed ragged array for the polygon nodes. In CF 2 we would expect the ragged array notation to change to a variable length data field that uses more natural 2d indexing.
-
-A piece of software reading this would be expected to:
-1) See CF-1.8 conventions
-2) See the timeSeries featureType
-3) Find the timeseries\_id cf\_role
-4) Find the coordinates 
-5) See that the coordinates have a standard\_name "polygon node"
-6) Find the variable with cf_role "node\_index" (is there another way to do this with ragged arrays? I'm typing from memory.)
-7) Iterate over stations, picking out geometries for each timeseries.
-
-Dimensionality;
-1) strlen is long enough for the character array timeseries id.
-2) node would be as long as all the nodes of all the timeseries + enough special characters to separate 'holes'. 
-3) timeseries would be as long as the number of timeseries in the file.
-4) index_start_stop would be 2 or 3 depending if just the first node of a given time series or both the first and last node of a given timeseries is to be stored.
-5) time is the length of the maximum length time series in the file. 
+## Data Elements and Structure
+
+For discussion, the structure and attributes below should be what's needed to store a polygon as the `timeseries_id` of a `timeSeries` featuretype. This is using a rectangular array for the time series data and a contiguous ragged array (indexed ragged array would be silly) for the polygon nodes. In CF 2 we would expect the ragged array notation to change to a variable length data field that uses more natural 2d indexing.
```

Comment:
In your example code, the variable with cf role timeseries_id is polygonsID, which is more akin to the polygon's name rather than its geometry.  I would reword this line to:
what's needed to store a polygon representing the geometry of a timeSeries feature within its instance variables.